app/BackgroundSync.ts

```diff
@@ -62,7 +62,7 @@ const BackgroundSync = async (task_data: any) => {
         // update batch_num with the new value, otherwise never change
         batch_num = ss.batch_num;
       }
-    }, 2000);
+    }, 5000);
```

Comment:
Is this in milliseconds?

Reply:
Yes, this only affect to the Sync Report Screen... you can see the info every 5 seconds, and I included a spin circle with 5 steps... IMO looks good, and the App response is better, plus Matilda doesn't care about this screen. I think you can see this effect in v120...